test/SILOptimizer/return.swift

```diff
@@ -213,3 +213,39 @@ func testSR13753() {
     //expected-warning@-1{{variable is unused}}
   }
 }
+
+// SR-14505
+struct SR14505 {
+    let b = true
+    var x: Int {
+        if b {
+            return 0
+        }
+    } // expected-error {{missing return in a getter expected to return 'Int'}}
+
+    var y: Int {
+        get {
+            if b {
+                return 0
+            }
+        } // expected-error {{missing return in a getter expected to return 'Int'}}
+        set {}
+    } 
+}
+
+class SR14505_C {
+  static let a = false
+  let b = true
+
+  func method() -> Int {
+    if b {
+      return 0
+    }
+  } // expected-error {{missing return in a instance method expected to return 'Int'}}
```

Comment:
Should this say `an instance method` or remove the article altogether?

Reply:
It make sense to remove the article, thanks @harlanhaskins :)